frontend/src/pages/_app.tsx

```diff
@@ -1,8 +1,27 @@
+import React from 'react'
 import '~/styles/globals.css'
 import type { AppProps } from 'next/app'
+import { Slide, ToastContainer } from 'react-toastify'
 
 const MyApp = ({ Component, pageProps }: AppProps) => {
-  return <Component {...pageProps} />
+  return (
+    <React.Fragment>
```

Comment:
be consistent when using fragment
```suggestion
    <>
```